app/mailers/user_mailer.rb

```diff
@@ -11,8 +11,17 @@ def work_order_notification(user, job, work_order)
     @user = user
     @job = job
     @work_order = work_order
+    subject = "#{@user.full_name}, you have been sent a work order for job #{@job.id}, #{@job.name}. Franchise: #{@job.franchise.try(:name)}."
 
-    mail(to: @user.email, subject: "#{@user.full_name}, you have been sent a work order for job #{@job.id}, #{@job.name}. Franchise: #{@job.franchise.try(:name)}.")
+    @mail = mail(to: @user.email, subject: subject)
+
+    MailLog.create(
+      name: @user.full_name,
+      email: @user.email,
+      subject: subject,
+      body: @mail.body.encoded,
+      mail_loggable: work_order
```

Comment:
I'm not sure if this will work. This should map to `mail_loggable_id` and `mail_loggable_type`, but work_order model doesn't have id or type columns. did this work for you?